tests/SnowflakeTest.php

```diff
@@ -218,6 +219,25 @@ public function testRunEmptyQuery(): void
         $app = $this->createApplication($config);
         $result = $app->run();
 
+        // Check query tags are present
+        sleep(30);
```

Comment:
Ty jo nejde to udělat nějak líp v nějakém loopu checkovat? Zabit to na 30s for sure, mě přijde docela brutal.

Reply:
Jo ono to tam v podstate byt nemusi, tak som to odelal a testy presli